legacy-docs/omnibox.md

```diff
@@ -3,6 +3,10 @@
 LabKey component that takes a set of actions (like filter, sort, search) and exposes them as a single input for applying those actions to a QueryGrid.
 
 ## Release Notes ##
+### version ???
+*Released*: ???
+* Package patch update for changes to @glass/base v???
```

Comment:
@labkey-susanh Not sure we care, but do you know why these legacy-docs were updated?

Reply:
No idea.  Probably the omnibox merge happened while this branch was open? (Yes, it was open for a long time.)